### ISSUE TITLE
Web Accessibility: ARIA trees not accessible with NVDA and Firefox or Chrome

### ISSUE THREAD

**Issue by behofer** [nvaccess/nvda#12012]
### Steps to reproduce:
1. Get ARIA tree code example (W3C recommendation) from W3C official website: https://www.w3.org/WAI/GL/wiki/Using_ARIA_trees
2. Copy Code Example into a plain HTML file
3. Open the HTML file from step 2 in Chrome or Firefox on Windows 10 with NVDA screenreader enabled
4. Navigate the tree with NVDA screenreader in browse mode

### Actual behavior:
NVDA will read the first tree element and its children (in the example "Fruits") properly whereas the following elements (example: "Vegetables") are not accessible at all/are skipped.
So, only information on the first tree element (Level 1) is passed to a screenreader.

### Expected behavior:
NVDA should be able to navigate the whole tree with all its elements in Chrome.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.3

#### Windows version:
Windows 10 Enterprise - Version	10.0.17763 Build 17763

#### Name and version of other software in use when reproducing the issue:
Firefox 79, Chrome 86
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviour? 
No.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by akash07k** [comment]
I confirm this bug.
Happens here with latest alpha as well on edge, chrome and firefox.

**Comment by Adriani90** [comment]
Could be related to #12350.

**Comment by Adriani90** [comment]
This is reproducible in all browsers, it seems even in focus mode the navigation in the test code is broken.

**Comment by Adriani90** [comment]
cc: @jcsteh is there something NVDA can do about it? Or is this something that need fixing in browsers themselves?
In focus mode the tree view is not navigable, and the only way to navigate really the whole tree with the keyboard is to use object navigation.

**Comment by SaschaCowley** [comment]
Aria treeviews **need** Javascript to operate correctly. Closing as invalid.

**Comment by Adriani90** [comment]
cc: @scottaohara is the part about java script covered in the ARIA specs for tree views at least?

@jcsteh I couldn't find a propper MDN documentation about this. Is there one which describes practically tested exacples? If ound only this one but it doesn't include any hints on how to make them accessible with java script:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tree_role

**Comment by jcsteh** [comment]
There's nothing special about trees. ARIA always has been just about how semantics are exposed. Specifying ARIA roles and properties has never magically provided keyboard functionality. This is just as true for a button as it is for a compound widget such as a tree.

To answer your question specifically, the ARIA spec (and MDN) both say that you need to manage focus for trees, among other roles. See https://w3c.github.io/aria/#managingfocus.
